.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -10,6 +10,13 @@ body:
       options:
         - label: I have searched the existing issues
           required: true
+  - type: input
+    attributes:
+      label: What version of Terraform are you using?
+      description: What version of Terraform are you using?
+      placeholder: 0.1.0-beta
```

Shouldn't beta releases also be numbered? E.g.

```suggestion
      placeholder: 0.1.0-beta1
```